<launch>
    <node pkg="arx_r5_controller" type="R5Controller" name="arm_vr" output="screen">
        <param name="arm_control_type" value="normal_v1"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_end_type" value="0"/>

        <param name="sub_topic_name" value="/ARX_VR_L"/>
        <param name="pub_topic_ee_name_v1" value="/arm_status_ee"/>
        <param name="pub_topic_joint_name_v1" value="/arm_status_joint"/>
    </node>
</launch>